tools/src/main/scala/com/nvidia/spark/rapids/tool/qualification/QualOutputWriter.scala

```diff
@@ -389,6 +395,14 @@ object QualOutputWriter {
   private[qualification] def getSummaryHeaderStringsAndSizes(
       appInfos: Seq[QualificationSummaryInfo],
       appIdMaxSize: Int): LinkedHashMap[String, Int] = {
+    val unSupportedExecsSum = appInfos.map(_.planInfo.map(_.execInfo.map(
+      _.unsupportedExecs.size).sum).sum)
+    val unSupportedExecsSize = appInfos.map(_.planInfo.map(_.execInfo.map(
+      _.unsupportedExecs.size).sum).sum).size
+    val unSupportedExprsSum = appInfos.map(_.planInfo.map(_.execInfo.map(
+      _.unsupportedExprs.flatten.size).sum).sum)
+    val unSupportedExprsSize = appInfos.map(_.planInfo.map(_.execInfo.map(
+      _.unsupportedExprs.flatten.size).sum).sum).size
```

Comment:
can you use flatMap instead of map and flatten?

Reply:
execInfo is an object where I am selecting unsupportedExprs( Array of String) and flattening it to get the total size. Please let me know if it could be improved further.